Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -250,50 +250,58 @@ Set-Alias -Scope Global Rebuild-Solution Invoke-RebuildSolution
 Function Invoke-RebuildSolution {
     Param(
         [string] $buildConfiguration = "Debug",
-        [string] $target = "build",
         [string] $verbosity = "minimal",
-        [string] $maxCpuCount = "",
```

Comment:
Sending an empty string for maxCpuCount would allow msbuild to use all available cores.

Reply:
Added it back